README.md

```diff
@@ -52,32 +59,32 @@ The Origin API is located on each server at `https://<host>:8443/oapi/v1`. These
 
 ### Kubernetes
 
-If you're looking for more information about using Kubernetes or the lower level concepts that Origin depends on, see the following:
+Since OpenShift Origin builds on Kubernetes, it is helpful to understand underlying concepts such as Pods and Replication Controllers. The following are good references:
 
-* [Kubernetes Getting Started](https://github.com/kubernetes/kubernetes/blob/master/README.md)
+* [Kubernetes User Guide](http://kubernetes.io/docs/user-guide/)
```

Comment:
Updated links to Kubernetes User Guide as referenced by the corresponding README in GitHub